### PR TITLE
Update tap-workload fragment to use new DSL

### PR DESCRIPTION
**Pull request by trisberg**

### CODE REVIEW

File: fragments/tap-workload/accelerator.axl [hunk 1]
```diff
@@ -0,0 +1,12 @@
+engine {
+  Exclude({"README.md"})
+  if(#bsGitRepository > "") {
```
Comment (ericbottard) [inline fragments/tap-workload/accelerator.axl:3]:
```suggestion
  if(#bsGitRepository != "") {
```

File: fragments/tap-workload/accelerator.axl [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    let repoUrl = T(org.springframework.web.util.UriComponentsBuilder).fromUriString('https://' + #bsGitRepository).build(true) in
+    {
+      applyTo("**/workload.yaml" || "**/workload-native.yaml" || "**/workloads.yaml" ) {
+        ReplaceText(regex: {pattern: "(?m)url: .*$", with: 'url: ' + #repoUrl.scheme + '://' + #repoUrl.host + '/' + #repoUrl.queryParams['owner'][0] + '/' + #repoUrl.queryParams['repo'][0]})
```
Comment (ericbottard) [inline fragments/tap-workload/accelerator.axl:7]: Why go to regex instead of OpenRewriteRecipe?

Comment (ericbottard) [inline fragments/tap-workload/accelerator.axl:7]: What's the point of using `(?m)` if you don't use `$` or `^` ?

Comment (trisberg) [reply fragments/tap-workload/accelerator.axl:7]: Much cheaper in terms of processing time, not sure why we had one recipe and one regex originally, should be the same for both I would think

File: fragments/tap-workload/accelerator.axl (hunk 1 above)
Comment (trisberg) [reply fragments/tap-workload/accelerator.axl:3]: So, for this I think we should do one of these:

```
        ReplaceText(regex: {pattern: "(?m)url: .*$", with: 'url: ' + #repoUrl.scheme + '://' + #repoUrl.host + '/' + #repoUrl.queryParams['owner'][0] + '/' + #repoUrl.queryParams['repo'][0]})
        ReplaceText(regex: {pattern: "(?m)branch: .*$", with: 'branch: ' + #bsGitBranch})
```

or 

```
        OpenRewriteRecipe(recipe: "org.openrewrite.yaml.ChangeValue", options: {oldKeyPath: "$.spec.source.git.url", value: #repoUrl.scheme + '://' + #repoUrl.host + '/' + #repoUrl.queryParams['owner'][0] + '/' + #repoUrl.queryParams['repo'][0]})
        OpenRewriteRecipe(recipe: "org.openrewrite.yaml.ChangeValue", options: {oldKeyPath: "$.spec.source.git.ref.branch", value: #bsGitBranch})
```

The OpenRewrite one takes significantly longer to process

Comment (trisberg) [reply fragments/tap-workload/accelerator.axl]: Using `if(#bsGitRepository != "")` gives this error from the engine

```
"org.springframework.expression.spel.SpelEvaluationException: EL1012E: Cannot index into a null value"
```

Comment (ericbottard) [reply fragments/tap-workload/accelerator.axl:3]: Well, indeed the original YAML version compared to `null`, not empty string

**Pull request merged**